src/components/data/pagination/components/RowsPerPage/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { Fragment } from 'react';
+
+const perPageOptions = [5, 10, 20, 50, 100];
```

Comment:
Could this be a type? E.g. `type PerPageOptions = 5 | 10 | 20 | 50 | 100;` That way users of the pagination component (and DataTable or whatever uses the pagination internaly) would get intellisense on the available per page options and couldn't pass `543` or whatever as per per page